### PR TITLE
Update ru_RU.lang

### PR DESCRIPTION
**Pull request by Adaptivity**
Also added pack.mcmeta

### CODE REVIEW

**Comment by Adaptivity** [comment]
pack.mcmeta not autogenerated. It should be created manually if it not created fml will print message like "missing a pack.mcmeta file" in log

**Comment by Adaptivity** [comment]
Latest build: http://i.imgur.com/exYXNSL.jpg

**Comment by Parker8283** [comment]
It IS autogenerated, and is logged in the debug channel, so it won't appear in the users console. Since the FML one is essentially the same as the one in your PR, I see no point.

**Comment by Adaptivity** [comment]
Many mods have added pack.mcmeta by hand. Look for TinkersConstruct, Ender IO etc. So why not in  IguanaTweaks? If it really bother you I'll revert it.

File: src/main/resources/assets/IguanaTweaks/lang/en_US.lang [hunk 1]
```diff
@@ -8,4 +8,4 @@ msg.bedGone=You awake to find your bed smashed to pieces
 hud.weight=Weight
 hud.encumberance.greatly=Greatly encumbered
 hud.encumberance.average=Encumbered
-hud.encumberance.slightly=Slightly encumbered
+hud.encumberance.slightly=Slightly encumbered
```
Comment (Parker8283) [inline src/main/resources/assets/IguanaTweaks/lang/en_US.lang]: add back the new line at the end of the file

File: src/main/resources/assets/IguanaTweaks/lang/ru_RU.lang [hunk 2]
```diff
@@ -4,3 +4,8 @@ msg.decorativeBeds=Никто не говорил Вам? Кровати про
 msg.setNoSleep=Ваше место появления было установлено, наслаждайтесь ночью
 msg.randReSpawn=Вы пришли в сознание и озадачены тем, где Вы находитесь
 msg.bedGone=Вы проснулись и обнаружили, что Ваша кровать разбита вдребезги
+
+hud.weight=Вес
+hud.encumberance.greatly=Вы сильно перегружены
+hud.encumberance.average=Вы перегружены
+hud.encumberance.slightly=Вы немного перегружены
```
Comment (Parker8283) [inline src/main/resources/assets/IguanaTweaks/lang/ru_RU.lang]: new line at the end of the file please

File: src/main/resources/pack.mcmeta [hunk 3]
```diff
@@ -0,0 +1,6 @@
+{
+  "pack": {
+    "pack_format": 1,
+    "description": "Resources used for Iguana Tweaks"
+  }
+}
```
Comment (Parker8283) [inline src/main/resources/pack.mcmeta]: same here

**Comment by Parker8283** [comment]
bah whatever just leave it. After fixing all that, squash please.

**Pull request closed without merging**